proposals/254-padding-negotiation.txt

```diff
@@ -63,6 +63,12 @@ parameters:
     u16 ito_high_ms;
   };
 
+After the above cell is received, the guard should use the 'ito_low_ms' and
+'ito_high_ms' values as the minimum and maximum values (respectively) for
+inactivity before it decides to pad the channel. The actual timeout value is
+randomly chosen between those two values through an appropriate probability
+distribution (see proposal251 for the netflow padding protocol).
+
 More complicated forms of link-level padding can still be specified
 using the primitives in Section 3, by using "leaky pipe" topology to
```

Comment:
If more complicated forms of link-level padding can be specified using the leaky pipe topology, why do link-level padding this way in the first place?

Reply:
Because the link-level padding is super-low overhead, and only protects against adversaries that are performing netflow data retention at large routers.

Reply:
I added a paragraph for this in 1dd7f1ff78fcb69121130cbd802b0b8b527ffc63

Reply:
https://github.com/torproject/torspec/pull/41/commits/1dd7f1ff78fcb69121130cbd802b0b8b527ffc63